src/content/structured/get-started/releases-versioning.mdx

```diff
@@ -12,11 +12,11 @@ contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/struc
 
 ## How we release and version
 
-This page provides details on how the `@ukic/` packages and website will be released and versioned.
+This page provides details on how the `@ukic/` packages and website are released and versioned.
 
 <p>
-  Releases will be triggered once the `develop` branch pull request is merged
-  into the `main` branch. The package versions will follow the{" "}
+  Releases are triggered once the `develop` branch pull request is merged
```

Comment:
On the website, the back ticks are shown for `main` and `develop` and I believe they should have the same formatting as `@ukic/` above - is there a reason they aren't rendered correctly? There are more instances of this throughout the site.

Reply:
I don't know why they aren't being formatted properly (for other reviewers, look at the [Releases and Versioning page](https://design.sis.gov.uk/get-started/releases-versions)) - maybe they're somehow not really backticks? I'd cheat, and copy-paste the characters around @ukic/ and paste them next to main and develop and see if that fixes it

Reply:
I think the backticks potentially aren't working because they're in a `<p>`, I think in other cases of this we use `<code>` to replace the backticks